_store/district09-archive-timesheets-command-bundle.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Archive timesheets command plugin for Kimai
+name: Archive timesheets command plugin
+intro: "A plugin, which allows you to archive timesheets older than a specified timeframe, using a command."
+developer: district09
+date: "2022-01-07 12:00:00 +0200"
+icon: fas fa-terminal
+price: "0"
+download: "https://github.com/digipolisgent/kimai_plugin_archive-timesheets-command/archive/master.zip"
+github: "https://github.com/digipolisgent/kimai_plugin_archive-timesheets-command"
+type: plugin
+tags:
+  - plugin
+bundle:
+  name: "ArchiveTimesheetsCommandBundle"
+  clone: "https://github.com/digipolisgent/kimai_plugin_archive-timesheets-command.git"
+---
+
+A Kimai 2 plugin, which allows you to archive timesheets older than a specified timeframe, using a command.
+
```

Comment:
Could you please add a sentence what "archiving timesheets" really means?

Archiving sounds as if you can still retrieve them from an archive, but in reality they are deleted.
Not that users expect a real archive and suddenly  experience a data loss.